addons/mrp/models/mrp_production.py

```diff
@@ -698,6 +711,26 @@ def _get_ready_to_produce_state(self):
             return 'assigned'
         return 'confirmed'
 
+    def action_view_mto_mrp_production_childs(self):
+        self.ensure_one()
+        return {
+            'name': _('MO Source of %s' % self.name),
+            'domain': [('id', 'in', self.procurement_group_id.stock_move_ids.created_production_id.ids)],
```

Comment:
https://github.com/odoo/odoo/pull/43366#issuecomment-575114872
The same

Reply:
The same